lib/Dialect/TTNN/Pipelines/TTNNPipelines.cpp

```diff
@@ -107,8 +112,16 @@ void createTTNNPipelineDeallocPassFromString(OpPassManager &pm,
   createTTNNPipelineDeallocPass(pm, *optionsStruct);
 }
 
+void createTTIRReshapeFoldPassFromString(OpPassManager &pm,
+                                         std::string options) {
+  auto optionsStruct =
+      TTIRToTTNNBackendPipelineOptions::createFromString(options);
+  createTTNNPipelineDeallocPass(pm, *optionsStruct);
+}
+
 void createTTIRToTTNNBackendPipeline(
     OpPassManager &pm, const TTIRToTTNNBackendPipelineOptions &options) {
+  createTTIRReshapeFoldPass(pm, options);
```

Comment:
Maybe this pass should live in `createTTNNPipelineTTIRPasses`?

@sdjordjevicTT can you comment on this? I'm not super familiar with the organization of these passes, but it seems to me that `createTTNNPipelineTTIRPasses` would be a better place for this pass?

Reply:
@sdjordjevicTT Did you have the time to take a quick look at this?